Bug 1196624 - Need official and unofficial 336 pixel Firefox branding…

danhuang1202 opened the pull request:
… icon for Firefox OS TV, r=rexboy

*add official/browser_336.png and unofficial/browser_336.png
*replace tv browse app branding icon from browser_284.png to browser_336.png

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3fea714908ac85ffc110b32a9409baa502585a47

rexboy7 commented on tv_apps/browser/index.html:
```diff
@@ -15,6 +15,9 @@
     <link rel="stylesheet" type="text/css" href="style/settings.css">
 
+    <!-- Resources -->
+    <link rel="resource" type="/image/png" href="shared/resources/branding/browser_336.png">
```
`type="image/png"`

danhuang1202 committed:
Bug 1196624 - Need official and unofficial 336 pixel Firefox branding icon for Firefox OS TV, r=rexboy

*add official/browser_336.png and unofficial/browser_336.png
*replace tv browse app branding icon from browser_284.png to browser_336.png

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1cd05bdd5f99b2871614f92f5e53b2b13712175b

danhuang1202 commented:
update type="image/png"
